### PR TITLE
Add loading screen component

### LINKED ISSUE #69: FE Useful Components - Loading Screens
**Is your feature request related to a problem? Please describe.**
Loading screens help indicate to the user that the app is doing something, improving responsiveness (as opposed to just rendering nothing)

**Describe the solution you'd like**
Loading components

### Checklist:

- [x] I have double checked that the issue does not currently exist. 
- [x] I have added the appropriate labels (e.g. feature, front-end, back-end).
- [x] I have added the **Platform Project** project

### PR DESCRIPTION
**Pull request by frasermcc9**
# Description
A loading screen helps make the app feel more complete, and hides things like auth loading.

Fixes/resolves #69 

## Screenshots
(its animated)
image

## Type of change
Please delete options that are not relevant.
- [X] **New feature** (non-breaking change which adds functionality)

# Checklist:
Leave blank if not applicable

I have completed these steps when making this pull request:
- [x] I have assigned my name to the issue
- [x] I have moved the issue to the **In Progress** column
- [x] I have labelled the PR appropriately
- [x] I have assigned myself to the PR

Before opening the PR for review:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have moved the linked issue to the **Review in Progress** column

### CODE REVIEW

**Review by FriedhelmWS — DISMISSED** [review]
LGTM

File: packages/frontend/src/components/HouseConflictModal.tsx [hunk 1]
```diff
@@ -1,54 +1,47 @@
...
+interface HouseConflictModalProps {
+  visible: boolean;
+  setVisible(value: boolean): void;
+}
+
+const HouseConflictModal: React.FC<HouseConflictModalProps> = ({
+  visible,
+  setVisible,
+}) => {
+  // ==============================================
```
Comment (FriedhelmWS) [inline packages/frontend/src/components/HouseConflictModal.tsx:13]: Is this comment necessary?

**Review by RahulA2000 — APPROVED** [review]
LGTM

File: packages/frontend/src/components/HouseConflictModal.tsx (hunk 1 above)
Comment (frasermcc9) [reply packages/frontend/src/components/HouseConflictModal.tsx:13]: This files being removed in another PR

**Review by j-chad — APPROVED** [review]
LGTM

**Pull request merged**